cg/meta/workflow/rnafusion.py

```diff
@@ -158,12 +159,8 @@ def get_references_path(self, genomes_base: Optional[Path] = None) -> Path:
     def get_default_parameters(self, case_id: str) -> Dict:
         """Returns a dictionary with default RNAFusion parameters."""
         return {
-            "input": NextflowAnalysisAPI.get_input_path(
-                case_id=case_id, root_dir=self.root_dir
-            ).as_posix(),
-            "outdir": NextflowAnalysisAPI.get_outdir_path(
-                case_id=case_id, root_dir=self.root_dir
-            ).as_posix(),
+            "input": self.get_case_path(case_id=case_id).as_posix(),
+            "outdir": self.get_case_path(case_id=case_id).as_posix(),
             "genomes_base": self.get_references_path().as_posix(),
             "trim": RnafusionDefaults.TRIM,
             "fastp_trim": RnafusionDefaults.FASTP_TRIM,
```

Comment:
same as above, but arguable too much to change when if moving away from dict to models, maybe something for the future

Reply:
in a following PR